client/sites-dashboard/components/no-sites-message.tsx

```diff
@@ -112,7 +112,7 @@ export const NoSitesMessage = ( { status }: SitesContainerProps ) => {
 				"It's time to get your ideas online. We'll guide you through the process of creating a site that best suits your needs."
 			) }
 			action={ __( 'Create your first site' ) }
-			actionURL={ '/start?ref=sites-dashboard' }
```

Comment:
I suspect `ref` may have some significance for analytics. It would be good to confirm one way or the other.

Reply:
@danielbachhuber I saw we used `source` in other conditions so I thought `ref` might be a mistake or just a placeholder there.

@p-jackson when you used `ref` in this link, was there a particular reason to do so? Do we use it for tracking referrals somehow?

Reply:
Could we use both `ref` and `source` here? `ref` being `calypso-nosites` on this page and `source` being `sites-dashboard`?

Reply:
You're bang on, `ref` is a tracking thing, although that's good digging @vykes-mac, I wasn't aware one of the onboarding steps was using it to set the default `write & publish` goal.

@zaguiini's suggestion sounds right, using both params. The `source` one is just about correctly setting the back URL, where as `ref` is for analytics purposes and we want the freedom to set it to whatever we want. Maybe the `EmptyContent` and button in the top right of the dashboard should have the same `ref`, or maybe they should be different. We want the freedom to do either.

Reply:
Thanks for the research. I've updated buttons and used `calypso-nosites` on empty sites block, and `sites-dashboard` in the top button. This way, we handle a go-back action properly and still have the freedom of using the `ref` parameter for tracking purposes.